tensorflow_addons/seq2seq/attention_wrapper.py

```diff
@@ -1499,14 +1483,13 @@ def hardmax(logits, name=None):
     Returns:
       A batched one-hot tensor.
     """
-    with ops.name_scope(name, "Hardmax", [logits]):
-        logits = ops.convert_to_tensor(logits, name="logits")
-        if tensor_shape.dimension_value(logits.get_shape()[-1]) is not None:
-            depth = tensor_shape.dimension_value(logits.get_shape()[-1])
+    with tf.name_scope(name or "Hardmax"):  #, [logits]):
+        logits = tf.convert_to_tensor(logits, name="logits")
+        if tf.compat.dimension_value(logits.get_shape()[-1]) is not None:
```

Comment:
did u miss v1 here?

Reply:
v1 and v2 are aliases for the same `tf.compat.dimension_value`